targets/maixbit.ld

```diff
@@ -0,0 +1,11 @@
+
+MEMORY
+{
+    RAM (xrw) : ORIGIN = 0x80000000, LENGTH = (6 * 1024 * 1024)
+}
+
+REGION_ALIAS("FLASH_TEXT", RAM);
```

Comment:
This seems to indicate that flash and RAM are in the same region. How does that work exactly? Is there a bootloader that loads the entire binary into RAM at reset?

Reply:
There is no integrated flash in the K210 chip. Exactly, the executable is flashed onto an external SPI flash and is copied at reset by a bootloader stored on the ROM to the RAM. I added the alias to keep the `riscv.ld` file unchanged.

Reply:
Right. That's not very common for baremetal systems, hence my confusion. Thanks for explaining.